amti/utils/workers.py

```diff
@@ -0,0 +1,40 @@
+""" Module for worker management functions """
+import boto3
+import click
+import csv
+from typing import List, Optional
+
+def create_batches(items: List, n=100) -> List:
```

Comment:
`create_batches` might be easy to confuse with the `create_batch` command, which can be avoided by naming this function something like `batch_list` or `chunk_list`.

Similarly, for the `n` argument, naming it `batch_size` and having no default forces people to explicitly pass it and could make the loops much more readable. Since the right batch size varies a lot from situation to situation, it's probably worth forcing callers to explicitly consider it.

Lastly, since the function is pretty general it might be better to put it in a new module (`amti.utils.misc` for example).